src/messages/layouts/MessageLayout.cpp

```diff
@@ -63,7 +63,7 @@ int MessageLayout::getHeight() const
 
 // Layout
 // return true if redraw is required
-bool MessageLayout::layout(int width, float scale, MessageElementFlags flags)
+bool MessageLayout::layout(int width, float scale, MessageElementFlags msgflags)
```

Comment:
Why was this renamed?
Apart from that it breaks the `camelCase` naming convention by being all lowercase.

Reply:
As the commit message says:

`warning C4458: declaration of 'flags' hides class member`

I'm happy to rename it. What should it be? msgFlags?